src/Network/Network/NetworkWatcher/ConnectionMonitor/SetAzureNetworkWatcherConnectionMonitorCommand.cs

```diff
@@ -79,178 +102,404 @@ public class SetAzureNetworkWatcherConnectionMonitorCommand : ConnectionMonitorB
         [Alias("ConnectionMonitorName")]
         [Parameter(
             Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
```

Comment:
You should not add ValueFromPipelineByPropertyName here

Reply:
fixed